### PR TITLE
feat: new media upload pipeline (only merge when backend changes are live on production)

### PR DESCRIPTION
**Pull request by hirbod**
# Why

This PR brings our new media upload pipeline to life!

# How

From now on, we will be uploading all of our media directly to our server and processing them on GCP before uploading them to IPFS/Bunny. Due to GCP's 32 MB limit, we have temporarily reduced the upload amount to 30 MB until we find a solution. However, note that 50 MB for IPFS is already a limiting factor.

# Test Plan

### CODE REVIEW

**Comment by hirbod** [comment]
@alantoa @intergalacticspacehighway DO NOT MERGE until @jorgeavaldez released to prod!

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️ (once API is up 😅 )

**Review by alantoa — APPROVED** [review]
LGTM

**Pull request merged**
